code/lib/cli/src/repro-generators/configs.ts

```diff
@@ -285,3 +285,11 @@ export const svelteKit: Parameters = {
   generator:
     'yarn create svelte-with-args --name={{appName}} --directory=. --template=skeleton --types=null --no-prettier --no-eslint --no-playwright',
 };
+
+export const nuxt_typescript: Parameters = {
```

Comment:
I believe we don't use this file anymore!

Reply:
yes i guessed it